incubator/command-dump/src/main/java/io/aklivity/zilla/runtime/command/dump/internal/airline/LoggableStream.java

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
No longer used? If so please remove.

Reply:
It's still used. I'd keep it, since this class wraps objects/methods which are needed to read from a stream, and the code is more readable this way.
But if you think, it's not well aligned with the other commands, I can move these out to the ZillaDumpCommand class.